ui/components/Header.js

```diff
@@ -63,6 +67,41 @@ const styles = (theme) => ({
 });
 
 class Header extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      meshSyncStatusSubscription : null,
```

Comment:
@sudo-NithishKarthik, @Abhishek-kumar09  - - just a brief note that the state of these objects will need to be accounted for during the UI refactoring - with redux.